wifiphisher/common/phishinghttp.py

```diff
@@ -19,13 +20,24 @@ def get(self, url):
         self.redirect("http://10.0.0.1:8080/")
 
 
-class ShowPhishingPageHandler(tornado.web.RequestHandler):
+class CaptivePortalHandler(tornado.web.RequestHandler):
```

Comment:
At some point, we want Wifiphisher to be able to show the phishing page while the victim has already browsed the Internet for a while. How about `PhishingSiteHandler`?